heartbeat/_meta/fields.common.yml

```diff
@@ -63,6 +63,21 @@
           description: >
             A token unique to a simultaneously invoked group of checks as in the case where multiple IPs are checked for a single DNS entry.
 
+        - name: next_run
```

Comment:
I normally stumble over names which are `foo` and `foo_bar` as it seems `foo` should probably be an object because the two are somehow related. I guess this is the case here, but not sure what I should recommend as a name ...

BTW: 100% personal preference.

Reply:
That's a good point. I think we only need `next_run_in`. This is in support of timelines in the future. Let me leave this PR open for now, when we get the Kibana side of this going and we're sure this does everything we need I can come back to this and merge it.

Reply:
SGTM. Ping me when I should have a look again.